buildpipeline/dotnet-standard-win.json

```diff
@@ -4,75 +4,70 @@
       "enabled": true,
       "continueOnError": false,
       "alwaysRun": false,
-      "displayName": "Install Signing Plugin",
+      "displayName": "git clone",
       "timeoutInMinutes": 0,
       "task": {
-        "id": "30666190-6959-11e5-9f96-f56098202fef",
+        "id": "d9bafed4-0b18-4f58-968d-86655b4d2ce9",
         "versionSpec": "1.*",
         "definitionType": "task"
       },
       "inputs": {
-        "signType": "$(SignType)",
-        "zipSources": "false",
-        "version": "1.0.268",
-        "feedSource": "https://devdiv.pkgs.visualstudio.com/DefaultCollection/_packaging/MicroBuildToolset/nuget/v3/index.json"
+        "filename": "$(PB_Git)",
+        "arguments": "clone $(PB_VsoCorefxGitUrl) standard",
```

Comment:
`PB_VsoCorefxGitUrl` doesn't seem like a good name (`Corefx` specifically), and is it defined anywhere?